src/adapters/communities-db.ts

```diff
@@ -7,5 +19,123 @@ export function createCommunitiesDBComponent(
 
   const logger = logs.getLogger('communities-db-component')
 
-  return {}
+  return {
+    async getCommunity(id: string, userAddress: string) {
```

Comment:
can we add returning types?

Reply:
Isn't it enough with the `ICommunitiesDatabaseComponent` type?